packages/lesswrong/components/ea-forum/giving-portal/hooks.ts

```diff
@@ -36,6 +36,8 @@ export const useElectionCandidates = (
       sortBy: sortBy === "random" ? "name" : sortBy,
     },
     limit: 30,
+    // There is an SSR mismatch bug that occurs on safari when using the random sort
+    ssr: false,
```

Comment:
I ran into [this bug](https://github.com/ForumMagnum/ForumMagnum/pull/8341) again, both safari and firefox give a different sorting on the frontend than what is generated on the server (whereas chrome matches the server). I'm now quite sure that it's just an SSR mismatch bug due to the random sorting, and it may or may not be the case that it's due to a genuine bug in seedrandom vs something else (e.g. the key we use being different somehow).

Because this is quite a serious bug if it happens (because you can accidentally vote for the wrong candidate) I decided no-ssr-ing is the safest thing to do for now, even though it adds some loading time (and some unfortunate flickering)